src/features/TransactionTable/TransactionTable.utils.ts

```
@@ -1,5 +1,11 @@
-export const formatCurrency = (amount: number, symbol: string = '$') =>
-  amount < 0 ? `-${symbol}${Math.abs(amount)}` : `${symbol}${amount}`;
+import { formatToCompactUnit } from '@utils/common.utils';
+
+export const formatCurrency = (amount: number, symbol: string = '$') => {
```

Biome (1.9.4)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)